sw/device/sca/aes_serial.c

```diff
@@ -39,6 +38,34 @@ enum {
   kIbexAesSleepCycles = 400,
 };
 
+/**
+ * Max number of encryption that can be captured with the scope
+ * 81 is selected for AES with CW Husky
+ * Note: Maybe it would be better if we use dynamic memory allocation but I
+ * am not sure whether we are supporting it or not.
+ */
+#define MAX_NUM_OF_OPS_IN_A_BATCH 81
+
+/**
+ * An array of keys to be used in a batch
+ */
+static uint8_t batch_keys[MAX_NUM_OF_OPS_IN_A_BATCH][kAesKeyLength];
+
+/**
+ * An array of plaintexts to be used in a batch
+ */
+static uint8_t batch_plaintexts[MAX_NUM_OF_OPS_IN_A_BATCH][kAesTextLength];
+
+/**
+ * Key selection between fixed and random key during the batch capture.
+ */
+static bool sample_fixed = true;
```

Comment:
I can't see any references to this outside `aes_serial_fvsr_key_batch_encrypt()`. Why is this a `static`?

Reply:
No reason actually :) I am changing it.